Userland/Utilities/js.cpp

```diff
@@ -738,7 +738,8 @@ static void print_value(JS::Value value, HashTable<JS::Object*>& seen_objects)
             return print_function(object, seen_objects);
         if (is<JS::Date>(object))
             return print_date(object, seen_objects);
-        if (is<JS::Error>(object))
+        auto prototype_or_error = object.internal_get_prototype_of();
+        if (is<JS::Error>(object) || (prototype_or_error.has_value() && prototype_or_error.value() == object.global_object().error_prototype()))
```

Comment:
`print_object` at the end might crash if we call it with a stored exception - also, can we only do the prototype lookup for non-`JS::Error` objects?

Reply:
ugh, I keep forgetting that we still throw normal exceptions even with ThrowCompletionOr, fixed.